modules/c/pages/_partials/downloadslist.adoc

```diff
@@ -47,25 +38,19 @@ ifndef::our-vs-version-hyphenated[:our-vs-version-hyphenated: x]
 :release-dirs-lib: pass:q,a[`{release-dir-lib}` or `{release-dir-ee-lib}`]
 :release-dirs: pass:q,a[`{release-dir}` or `{release-dir-ee}`]
 
+// If both parameters are defined, render CBL downloads
```

Comment:
I don't understand what you're doing with the ifdefs here and above - I assume you've tested and it works OK, but if you'd like any sense checking from me we should jump on a call so you can explain it to me :D